model/resource/deployment_environment.yaml

```diff
@@ -10,4 +10,12 @@ groups:
         brief: >
           Name of the [deployment environment](https://wikipedia.org/wiki/Deployment_environment)
           (aka deployment tier).
+        note: >
+          `deployment.environment` does not affect the uniqueness constraints defined through
```

Comment:
Hi @bripkens 
You need to re-generate the markdowns after changing the YAML here. See here https://github.com/open-telemetry/semantic-conventions/blob/main/model/README.md#generating-markdown

Reply:
Thank you @joaopgrassi, done